src/components/MobileDropdown.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+export default function MobileDropdown() {
+  return (
+    <ul className="absolute left-0 top-[96px] bg-gray-400 w-full flex flex-col items-center gap-1 py-1 text-xl font-normal text-gray-300 lg:hidden">
+      <li className="hover:text-white hover:underline w-full py-2">
+        <Link
+          onClick={() => {
+            setMenu(false);
+          }}
+          className="w-full flex justify-center"
+          to="/"
+        >
+          Home
+        </Link>
+      </li>
+      <div className="border w-full" />
+      <li className="hover:text-white hover:underline w-full py-2">
+        <Link
+          onClick={() => {
+            setMenu(false);
```

Comment:
These setMenu() calls do not do anything as the function is not being imported into this component. But the menu resets anyways because we navigate to a different page.